test-utils/src/main/java/com/criteo/publisher/TestDeviceIds.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ *    Copyright 2020 Criteo
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.criteo.publisher
+
+const val NON_CONSENTED_DEVICE_ID = "2"
```

Comment:
Nit: the project already contains the `StubConstants` class containing constants.
I think that this file will not contains anything but this line. What do you think about moving this into `StubConstants` instead ?

Reply:
Well, StubConstants contains .. stub constants which isn't the case of this deviceId. I feel that even if there is only a single constant in this file, it is still worth separating concerns and having clear boundaries. Or we should rename `StubConstants` into `TestConstants` or something like that.

Reply:
Renaming `StubConstants` will be a breaking change in lot of tests. Included ones of mediation adapters.
Ok to keep this file then. Maybe should you encapsulate the constant in an kotlin `object` or use `@file:JvmName` to make it nicer from Java code (see https://stackoverflow.com/a/57355475/3663982 to make your choice).